client/layout/logged-out-design.jsx

```diff
@@ -9,9 +9,11 @@ import React from 'react';
  * Internal dependencies
  */
 import MasterbarLoggedOut from 'layout/masterbar/logged-out';
+import ThemesHead from 'my-sites/themes/head';
 
-const LayoutLoggedOutDesign = () => (
+const LayoutLoggedOutDesign = ( { tier = 'all' } ) => (
 	<div className="wp is-section-design has-no-sidebar">
+		<ThemesHead tier={ tier } />
```

Comment:
Aren't we rendering a head component both here, and in the controller, when logged out?